core/modules/taxonomy/taxonomy.module

```diff
@@ -1747,18 +1747,20 @@ function taxonomy_field_formatter_settings_summary($field, $instance, $view_mode
 /**
  * Returns the set of valid terms for a taxonomy field.
  *
- * @param $field
+ * @param array $field
  *   The field definition.
- * @return
+ * @param bool $indent
+ *   Whether to add indentation dots to the terms based on their depth.
+ * @return array
  *   The array of valid terms for this field, keyed by term id.
  */
-function taxonomy_allowed_values($field) {
+function taxonomy_allowed_values($field, $indent = TRUE) {
```

Comment:
We should add an `@since` for this parameter in the docblock of the function. We can leave it as a placeholder if we don't know the exact version when this'll get in, and the core committer that merges is can adjust the version number.

Reply:
Added.